torch/export/_trace.py

```diff
@@ -706,7 +706,7 @@ def _maybe_fixup_gm_and_output_node_meta(old_gm, new_gm):
         + len(graph_signature.buffers)
         + len(graph_signature.input_tokens)
     )
-    set_missing_meta_vals(gm, flat_fake_args, total_non_user_inputs, constant_attrs)
+    set_missing_meta_vals(gm, flat_fake_args, total_non_user_inputs)
```

Comment:
Hmm i wonder if the more correct thing to do is to beef up our constant attr finding logic? I guess it might be impossible if it is getting pulled from a method...

Reply:
Yeah I suspect so.